docs/.vitepress/vitepress/components/full-screen/vp-theme-toggler.vue

```diff
@@ -9,11 +9,13 @@ const locale = useNavbarLocale()
 <template>
   <div class="full-screen-theme-toggler">
     <span>Theme</span>
-    <CommonThemeToggler
-      :aria-label="locale['theme-toggler']"
-      :aria-checked="isDark"
-      @click="toggleDark()"
-    />
+    <ClientOnly>
+      <CommonThemeToggler
+        :aria-label="locale['theme-toggler']"
+        :aria-checked="isDark"
+        @click="toggleDark()"
+      />
+    </ClientOnly>
```

Comment:
Is `clientOnly` necessary here? 🤔

Reply:
I made some modifications. Please take a look to see if this approach might be better?

Reply:
> Is `clientOnly` necessary here? 🤔

isDark relies on the native browser API, so I wrapped it with `ClientOnly`